src/randoop/EmptyExceptionCheck.java

```diff
@@ -0,0 +1,57 @@
+package randoop;
+
+import randoop.sequence.Execution;
+
+/**
+ * An {@code ExceptionCheck} that doesn't enforce the expectation of an
+ * exception at a particular statement. Allows for execution at the statement 
+ * to be either normal or throw an exception.
+ */
+public class EmptyExceptionCheck extends ExceptionCheck {
+  
+  private static final long serialVersionUID = 8159839339961663076L;
+  
+  /**
+   * Creates an exception check at the given statement.
```

Comment:
More specifically, an EmptyExceptionCheck (or an empty exception check).
What is "the given statement"?  One isn't supplied, but `@param exception` mentions it too.  I would refer to statementIndex somehow.